cves/CVE-2011-1183.yml

```diff
@@ -116,7 +124,9 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
+  answer: | 
+    After searching the git log for the bug ID and following each of the links on cve.mitre.org, 
+    there was no evidence found of the vulerability discovery
```

Comment:
This just feels like some odd wording. While it works, I feel that it could be made clearer by wording it in another way.

Reply:
Maybe something along the lines of `"There was no information found about how the vulnerability was discovered in the git logs and on cve.mitre.org."`

Reply:
Agreed.